modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/components/BuilderScreen/BuilderScreen.tsx

```diff
@@ -151,9 +155,10 @@ export function BuilderScreen({
 												  )
 											: filter
 											? viewColumn?.objectFieldBusinessType
-											: viewColumn?.label[
-													defaultLanguageId
-											  ]
+											: getLocalizableLabel(
+													viewColumn?.label,
+													creationLanguageId!
+											  )
```

Comment:
@alinedoleron we can use the `viewColumn.objectFieldName` as the `fallback` here.